website/theme.config.tsx

```diff
@@ -2,9 +2,33 @@
 /* eslint sort-keys: error */
 import { useRouter } from 'next/router';
 import { defineConfig, Giscus, useTheme } from '@theguild/components';
+import { MendableSearchBar } from '@mendable/search';
 
 export default defineConfig({
   docsRepositoryBase: 'https://github.com/dotansimha/graphql-code-generator/tree/master/website',
+  search: {
+    component: () => {
+      return (
+        <div className="hidden w-[250px] sm:block">
+          <MendableSearchBar
```

Comment:
seems we can remove the outer div? and add className directly to MendableSearchBar
<img width="369" alt="image" src="https://github.com/dotansimha/graphql-code-generator/assets/7361780/77ca1042-4eb8-4638-9fbe-6fb7b5f87a55">